examples/hf-serverless-deployment.py

```diff
@@ -9,7 +9,8 @@
 from synthetic_dataset_generator import launch
 
 os.environ["HF_TOKEN"] = "hf_..."  # push the data to huggingface
-os.environ["MODEL"] = "meta-llama/Llama-3.1-8B-Instruct"  # use instruct model
+os.environ["MODEL"] = "meta-llama/Llama-3.1-8B-Instruct"  # use model for instruction generation
+os.environ["MODEL_COMPLETION"] = "meta-llama/Llama-3.1-70B-Instruct"  # use a different model for completions
```

Comment:
CAn you change this too a more explicit example. `different_model_for_completion.py` or something>